gh-skeleton

```diff
@@ -152,24 +160,21 @@ END_OF_LINE
     # Note that gh will attempt to add the remote origin and will throw an error
     # if it already exists.  We'll add the origin later if gh fails.
     remote_repo_status="created"
-    gh repo create --public --confirm "${dest_org}/${dest_repo}" || remote_repo_status="failed"
+    gh repo create --public "${dest_org}/${dest_repo}" || remote_repo_status="failed"
```

Comment:
Is there a reason you're not just using the new options to replicate previous behavior? Per the pull request you linked in the description:

> There are additional new flags:
>
>    `--source`: specify the path to the local git repo
>    `--remote`: specify the remote name to add (used with `--source`)
>    `--push`: push local commits to the new repo (used with `--source`)

Reply:
They updated the [`2.3.0` release notes](https://github.com/cli/cli/releases/tag/v2.3.0) to mention that change:

> Upload an existing local repository with
> 
> ```
> $ gh repo create myrepo --source=path/to/repo --public
> ```

Reply:
I've removed the "Note" comment that is no longer applicable.  

I didn't use the `--source`/`--remote` options because I think it makes more sense to configure the remotes when the repository is created locally, and not when `gh` does its thang:

https://github.com/cisagov/gh-skeleton/blob/244bda9e2414ad7083c5530ae142bca6a63d5452/gh-skeleton#L107-L111

I think this is more explicit, less magical, and hopefully shelters us from `gh` changing its behavior in the future.  Also, it will get the local repo in a good state sooner, so if we do have a failure there is a better change of the user being able to manually recover and continue.